tests/Rules/data/NoModelForwardingToBuilderInstance.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+/** @var \App\User $user */
+$user = \App\User::find(1);
+$user->first();
+$user->get();
+$user->find();
+$user->paginate();
+$user->where('id', 1);
+$user->take(1);
```

Comment:
On both of these tests, you need a method that gets forwarder to the QueryBuilder (not the EloquentBuilder), you should be able to use `$user->max('foo')` and `User::max('foo')` for this purpose